backend/src/schema/resolvers/groups.js

```diff
@@ -131,6 +133,76 @@ export default {
         session.close()
       }
     },
+    UpdateGroup: async (_parent, params, context, _resolveInfo) => {
+      const { categoryIds } = params
+      const { id: groupId, avatar: avatarInput } = params
+      delete params.categoryIds
+      if (CONFIG.CATEGORIES_ACTIVE && categoryIds) {
```

Comment:
What happens when categories are active but `categoriesIds` is `undefined`

Reply:
If `categoriesIds` are `null` or `undefined` they are not changed in any way.

If `categoriesIds` are `[]` an error is thrown if `CONFIG.CATEGORIES_ACTIVE === true`.
If the array is not empty the old are deleted and new are set.

But there's no type check if it's an array. You thing it's needed?

Reply:
The type check is done by GQL … 😉